### ISSUE TITLE
Is it possible to train with local image sets?

### ISSUE THREAD

**Issue by ra3njaran** [divamgupta/diffusionbee-stable-diffusion-ui#398]

**Comment by griffindavidson** [comment]
I don't believe DiffusionBee is designed to train your model. You'll need to train your model elsewhere and then download it and import it into DiffusionBee to use it. DiffusionBee is where you use your models, likely not train them

**Comment by impallari** [comment]
The world needs an app for easy training on Mac M1

**Comment by BrutalCoding** [comment]
@divamgupta has got you covered with https//liner.ai.

It's also mentioned at the top of the README in this repo.

Liner seems to be closed source, but other than that it's great. Fully offline.
